commit 3611376011c03472665d0144cb4c7a1b1d52d6ed (refs/changes/24/3924/3)
Author: Jason King <jason.king@joyent.com>
Date:   2018-05-09T18:17:41-05:00 (1 year, 5 months ago)
    
    OS-6943 varpd not listed as an install_h target
    OS-6946 varpd structs fail ctfdiff check

diff --git a/usr/src/lib/Makefile b/usr/src/lib/Makefile
index 7d337e7175..9efd87bc10 100644
--- a/usr/src/lib/Makefile
+++ b/usr/src/lib/Makefile
@@ -506,6 +506,7 @@ HDRSUBDIRS=				\
 	smbsrv		\
 	smhba		\
 	udapl		\
+	varpd		\
 	$($(MACH)_HDRSUBDIRS)
 
 i386_HDRSUBDIRS=	\
diff --git a/usr/src/lib/varpd/libvarpd/common/libvarpd_impl.h b/usr/src/lib/varpd/libvarpd/common/libvarpd_impl.h
index 7ecd3a952f..60f0dc5fff 100644
--- a/usr/src/lib/varpd/libvarpd/common/libvarpd_impl.h
+++ b/usr/src/lib/varpd/libvarpd/common/libvarpd_impl.h
@@ -100,6 +100,7 @@ typedef struct varpd_client_instance_arg {
 typedef struct varpd_client_nprops_arg {
 	uint64_t	vcna_id;
 	uint_t		vcna_nprops;
+	uint8_t		vcna_pad[4];
 } varpd_client_nprops_arg_t;
 
 typedef struct varpd_client_propinfo_arg {
@@ -109,6 +110,7 @@ typedef struct varpd_client_propinfo_arg {
 	uint_t		vcfa_prot;
 	uint32_t	vcfa_defsize;
 	uint32_t	vcfa_psize;
+	uint8_t		vcfa_pad[4];
 	char		vcfa_name[LIBVARPD_PROP_NAMELEN];
 	uint8_t		vcfa_default[LIBVARPD_PROP_SIZEMAX];
 	uint8_t		vcfa_poss[LIBVARPD_PROP_SIZEMAX];
diff --git a/usr/src/uts/common/sys/overlay_target.h b/usr/src/uts/common/sys/overlay_target.h
index cae193c334..ae92ef3532 100644
--- a/usr/src/uts/common/sys/overlay_target.h
+++ b/usr/src/uts/common/sys/overlay_target.h
@@ -282,6 +282,7 @@ typedef struct overlay_targ_cache_iter {
 	uint32_t			otci_pad;
 	uint64_t			otci_marker;
 	uint16_t			otci_count;
+	uint8_t				otci_pad2[3];
 	overlay_targ_cache_entry_t	otci_ents[];
 } overlay_targ_cache_iter_t;
 
